_analyzers/token-filters/kuromoji-completion.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+title: Kuromoji completion
+parent: Token filters
+nav_order: 230
+---
+
+# Kuromoji completion token filter
+
+The `kuromoji_completion` token filter is used to stem Katakana words in Japanese, which are often used to represent foreign words or loanwords. This filter is especially powerful for autocompletion or suggest queries, where partial matches on Katakana words can be expanded to include full forms.
+
+To use this token filter, you must install the `analysis-kuromoji` plugin on all nodes by running `bin/opensearch-plugin install analysis-kuromoji` and restart the cluster. For more information about installing additional plugins, see [Additional plugins]({{site.url}}{{site.baseurl}}/install-and-configure/additional-plugins/index/).
```

Comment:
```suggestion
To use this token filter, you must install the `analysis-kuromoji` plugin on all nodes by running `bin/opensearch-plugin install analysis-kuromoji` and restarting the cluster. For more information about installing additional plugins, see [Additional plugins]({{site.url}}{{site.baseurl}}/install-and-configure/additional-plugins/index/).
```

Reply:
"ing" does not work here because install uses the command and then you restart the cluster. Reworded.